### PR TITLE
Bump origin-web-common to 0.0.11

### PR DESCRIPTION
**Pull request by spadgett**
@dtaylor113 @jeff-phillips-18 fyi

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
     "patternfly-bootstrap-treeview": "2.1.1",
     "angular": "1.5.11",
     "kubernetes-object-describer": "1.0.4",
-    "angular-sanitize": "1.5.11"
+    "angular-sanitize": "1.5.11",
+    "origin-web-common": "0.0.11"
```
Comment (spadgett) [inline bower.json]: @jwforres we should be able to remove this resolution when we bump origin-web-catalog

**Comment by dtaylor113** [comment]
LGTM, you seem to have removed from console all items which had moved to common 0.0.8-0.0.11.  Not sure why the README changed.

**Comment by jwforres** [comment]
LGTM but travis failed

@dtaylor113 the README change was something sam and i discussed, it was an out of date list and most if not all of that is now in the common repo.

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
Fixed the dist problem

[merge]

**Pull request merged**